hedera-node/hedera-mono-service/src/main/java/com/hedera/node/app/service/mono/ServicesState.java

```diff
@@ -353,6 +354,8 @@ private ServicesApp internalInit(
                     .build();
             APPS.save(selfId, app);
         }
+        app.maybeNewRecoveredStateListener().ifPresent(listener -> platform.getNotificationEngine()
```

Comment:
Wouldn't we register this along with other listeners ??

Reply:
I'm not 100% sure that `SwirldMain.init()` is called during `EVENT_STREAM_RECOVERY`. So put it here to be sure it would be registered. We should ask...

Reply:
Oh okay !